op-e2e/e2eutils/geth/fakepos.go

```
@@ -26,6 +26,20 @@ type Beacon interface {
 	StoreBlobsBundle(slot uint64, bundle *engine.BlobsBundleV1) error
 }
 
+// FakePoS is a testing-only utility to attach to Geth
+// the exported struct is used by the devstack to control the fakePoS module, and adheres to the devstack.Lifecycle interface
+type FakePoS struct {
+	fakePoS *fakePoS
+}
+
+func (f *FakePoS) Start() {
+	_ = f.fakePoS.Start()
```

Suggestion: These errors get lost, and not logged when error. Currently L1CLNode does not have a logger field to do so. Shall we add a devtest.P field to L1CLNode and use this for log when error? Also, I think we need a separate sysgo struct for this: for example at sysgo/l1_nodes.go

```go
struct FakePoS {
    p devtest.P
    fakepos        *geth.FakePoS
}

func (f *FakePoS) Start() {
    f.p.Require(f.fakePoS.Start(), "fakePoS failed to start")
}


func (f *FakePoS) Stop() {
    f.p.Require(f.fakePoS.Stop(), "fakePoS failed to stop")
}
```
and plug in the sysgo fakePoS to sysgo L1CLNode.